src/view/uielement.js

```diff
@@ -112,6 +112,12 @@ function jumpOverUiElement( evt, data, domConverter ) {
 			const domOffset = domSelection.getRangeAt( 0 ).startOffset;
 
 			const viewPosition = domConverter.domPositionToView( domParent, domOffset );
+
+			// In case if dom element is not converted to view or is not mapped or something. Happens for example in some tests.
+			if ( viewPosition === null ) {
```

Comment:
Isn't this only a problem with these tests? How many tests were crashing on this?